index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <header><nav>
+        <a href="#" > Gul</a>
+        <a href="#" ><img src="images/Union.png"></a></nav>
+    </header>
+
+    <section id="top">
+        <h1> I’m Umarah Gul. Glad to see you!
+        </h1>
+        <p class="pg1">I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded, don’t
+            hesitate to contact me.</p>
+        <h2 class="connect">LET'S CONNECT</h2>
+        <ul class="footer">
+            <li><a href="https://www.twitter.com"><img src="images/twitter.png" /></a></li>
+            <li><a href="https://www.twitter.com"><img src="images/github.png" /></a></li>
+            <li><a href="https://www.twitter.com"><img src="images/Linkedin icon.png" /></a></li>
+            <li><a href="https://www.twitter.com"><img src="images/medium.png" /></a></li>
+            <li><a href="https://www.twitter.com"><img src="images/angellist.png" /></a></li>
+        </ul>
+    </section>
```

Comment:
Great job on your code! However, I noticed a few areas that could use some adjustments. 

- [ ] Firstly, the background image you've used doesn't align with the design. Please make sure to update your code to achieve the desired layout and positioning of the background image as specified in the design.

| Your | Professional |
| --- | --- |
| ![image](https://github.com/umarahgul/Portfolio/assets/55840999/d29c7f6d-2e35-4e6d-b951-23c7d81869e7) | ![image](https://github.com/umarahgul/Portfolio/assets/55840999/6eb4bf80-870d-4847-b7a7-f0c9a1a3e7a8) |

- [ ] Secondly, the gap between the header and section elements doesn't match the design. I recommend adjusting the spacing to match the specified gap and ensure consistency with the design requirements.

| Your | Professional |
| --- | --- |
| ![image](https://github.com/umarahgul/Portfolio/assets/55840999/c18bfced-2030-42ea-bbca-e9a25ade76d3) | ![image](https://github.com/umarahgul/Portfolio/assets/55840999/6eb4bf80-870d-4847-b7a7-f0c9a1a3e7a8) |

Keep up the good work, and don't forget to make these adjustments to align your code with the design specifications! 👍